### ISSUE TITLE
Compile fails on Windows10 x86-64, help or get me a pre-compiled installable

### ISSUE THREAD

**Issue by jeanflorentine** [zabirauf/icsharp#55]
Hi,
I am not a C# compilation dude, as my point is to use this C# notebook to train into this language,
unfortunately after some efforts I cannot compile the C# engine from sources on my Windows x86-64 laptop. Would be nice if there would be a precompiled installable of this C# Jupyter engine.
![screenshot 12-03-18 at 06 11 pm](https://user-images.githubusercontent.com/43513099/49389663-2a48cf00-f727-11e8-82e1-18fdc2807bbc.PNG)

**Comment by deepu2010** [comment]
Yes I too face this same issue, but it would be a great asset if C# works in jupyter notebook

**Comment by kirinnee** [comment]
1) Clone the git repo recursively to include the CSScript runtime
`git clone --recursive https://github.com/zabirauf/icsharp`

2) Navigate to the engine folder, and open ScriptCS.sln in VS2017

3) Change to Release from debug, Clean and rebuild solution (right click solution in solution explorer and rebuild twice works too)

4) Close VS2017 and go back to root and open iCSharp.sln, rebuild solution in Release

you will have the dll built

**Comment by mikail-gultekin** [comment]
@kirinnee, should promote your answer to the installation page.
